src/janitor.c

```diff
@@ -351,7 +350,7 @@ void per_loop_maint(void)
 		} else {
 			active_count += statlist_count(&login_client_list);
 		}
-		/* fallthrough */
+	/* fallthrough */
```

Comment:
This seems suboptimal. Let me see if I can fix that.

Reply:
Keeping the old indenting would be better, but I'm not too bothered by it either way.